conda_vendor/conda_lock_wrapper.py

```diff
@@ -0,0 +1,25 @@
+from conda_lock.src_parser.environment_yaml import parse_environment_file
+from conda_lock.conda_solver import solve_specs_for_arch, _reconstruct_fetch_actions, solve_conda
```

Comment:
oh wow we basically built some of this `_reconstruct_fetch_actions` :(

Reply:
lol, yeah as I was reading through the conda-lock src, it looked almost identical. I figured it'd be best to not duplicate work/code and just wrap this function as a static method .